doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-scenarios-background-instance-con.adoc

```diff
@@ -3,6 +3,17 @@
 
 In test scenario designer, you can use the *Background* tab to add and set the background data for rules-based as well as DMN based test scenarios. You can add and define the GIVEN data which is common for the entire test scenario simulation, based on the available data objects. *Background* tab has the ability to add and share the data among every test scenario. Data added using the *Background* tab can not be overridden by *Model* tab data.
```

Comment:
s/set the background data for/set background data for

Suggestion - simply sentence if possible and apply the hyphen to all instances of DMN based:

s/rules-based as well as DMN based/rules-based and DMN-based

Reply:
Done